tests/acceptance/fixtures/responses/universal.js

```diff
@@ -484,3 +523,15 @@ export const MockedUniversalSearchRequest = RequestMock()
     return urlRegex.test(request.url) && request.method === 'get';
   })
   .respond(UniversalSearchResponse, 200, CORSHeaders);
+
+export const MockedUniversalAutoCompleteRequest = RequestMock()
```

Comment:
would it make sense to put the universal search and universal autocomplete in separate files?

Reply:
Sure! I could make a `responses/universal` folder with two separate files.